source/keymap.js

```diff
@@ -12,7 +12,8 @@
 			19  : 'pause',
 			412 : 'rewind',
 			417 : 'fastforward',
-			461 : 'back'
+			461 : 'back',
+			8	: 'back'	//webkit keycode of backspace key in keyboard
```

Comment:
Hi @aarontam  Thank you for your opinion.
I think, it is not PC only. Some user can connect usb keyboard to TV.